src/main/java/io/strimzi/kafka/quotas/VolumeUsage.java

```diff
@@ -31,46 +31,47 @@ public VolumeUsage(String brokerId, String logDir, long capacity, long available
     }
 
     /**
-     *
+     * The source BrokerId.
      * @return The brokerId for the broker holding the volume
      */
     public String getBrokerId() {
         return brokerId;
     }
 
     /**
-     *
+     * The log directory, logDir in Kafka parlance.
      * @return the path identifying the logDir on the broker its hosted by.
      */
     public String getLogDir() {
         return logDir;
     }
 
     /**
+     * The capacity of the underlying Volume.
```

Comment:
Is this called from anywhere? If not should we remove it?

Reply:
It's not currently but I think is needed in the next PR adding metrics, which will include reporting available capacity.